src/normalize/entities.ts

```diff
@@ -415,11 +415,21 @@ const ContactAttributes = new Entity({
 	image: ImageFields
 });
 
+const CustomMetaAttributes = new Entity({
+	chatConversationId: 'chatConversationId'
```

Comment:
zimlet developers will need flexibility to define their own meta tags, so hardcoding here will not help them

Reply:
Done